package.json

```diff
@@ -31,6 +31,7 @@
     "mz": "^2.7.0",
     "netlify-lambda": "^1.4.13",
     "polished": "2.3.0",
+    "query-string": "^6.6.0",
```

Comment:
Don't think this is used anywhere else?

Reply:
Hm... you are right, it's not used anywhere. I'll remove it.